src/Miew.js

```diff
@@ -3444,6 +3444,12 @@ Miew.prototype._initOnSettingsChanged = function () {
     this._needRender = true;
   });
 
+  on('modes.VD.IsoMode', () => {
+    const mesh = this._getVolumeVisual().getMesh();
```

Comment:
What if there's no volume in the scene?

Reply:
Good comment, thanks, this case should be handled. Resolved now!